demos/ota/ota_demo_core_http/demo_config.h

```diff
@@ -0,0 +1,176 @@
+ * Settings/Custom Endpoint, or using the describe-endpoint API.
+ *
+ * #define AWS_IOT_ENDPOINT               "...insert here..."
+ */
+#define AWS_IOT_ENDPOINT    "a3e1obletr43ue-ats.iot.us-west-2.amazonaws.com"
+
+/**
+ * @brief AWS IoT MQTT broker port number.
+ *
+ * In general, port 8883 is for secured MQTT connections.
+ *
+ * @note Port 443 requires use of the ALPN TLS extension with the ALPN protocol
+ * name. When using port 8883, ALPN is not required.
+ */
+#define AWS_MQTT_PORT       ( 8883 )
+
+/**
+ * @brief AWS IoT Core server port number for HTTPS connections.
+ *
+ * For this demo, an X.509 certificate is used to verify the client.
+ *
+ * @note Port 443 requires use of the ALPN TLS extension with the ALPN protocol
+ * name being x-amzn-http-ca. When using port 8443, ALPN is not required.
+ */
+#ifndef AWS_HTTPS_PORT
+    #define AWS_HTTPS_PORT    443
+#endif
+
+
+/**
+ * @brief Path of the file containing the server's root CA certificate.
+ *
+ * This certificate is used to identify the AWS IoT server and is publicly
+ * available. Refer to the AWS documentation available in the link below
+ * https://docs.aws.amazon.com/iot/latest/developerguide/server-authentication.html#server-authentication-certs
+ *
+ * Amazon's root CA certificate is automatically downloaded to the certificates
+ * directory from @ref https://www.amazontrust.com/repository/AmazonRootCA1.pem
+ * using the CMake build system.
+ *
+ * @note This certificate should be PEM-encoded.
+ * @note This path is relative from the demo binary created. Update
+ * ROOT_CA_CERT_PATH to the absolute path if this demo is executed from elsewhere.
+ */
+#ifndef ROOT_CA_CERT_PATH
+    #define ROOT_CA_CERT_PATH    "certificates/AmazonRootCA1.crt"
+#endif
+
+#ifndef ROOT_CA_CERT_PATH_HTTP
+    #define ROOT_CA_CERT_PATH_HTTP    "certificates/BaltimoreCyberTrustRoot.crt"
+#endif
+
+/**
+ * @brief Path of the file containing the client certificate.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define CLIENT_CERT_PATH    "...insert here..."
+ */
+#define CLIENT_CERT_PATH           "/home/risc/work/iot/things/otalinux/cert.crt"
+
+/**
+ * @brief Path of the file containing the client's private key.
+ *
+ * Refer to the AWS documentation below for details regarding client
+ * authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define CLIENT_PRIVATE_KEY_PATH    "...insert here..."
+ */
+#define CLIENT_PRIVATE_KEY_PATH    "/home/risc/work/iot/things/otalinux/pri.key"
+
+/**
+ * @brief MQTT client identifier.
+ *
+ * No two clients may use the same client identifier simultaneously.
+ */
+#ifndef CLIENT_IDENTIFIER
```

Comment:
Suggest using the `MQTT` prefix to allow it to be distinguished from the HTTP connection used by the demo.
```suggestion
#ifndef MQTT_CLIENT_IDENTIFIER
```